index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Food lover</title>
+  <link rel="stylesheet" href="./styles/style.css"> 
+  <link rel="stylesheet" href="./styles/index.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+</head>
+<body>
+    <header>
+      <img class="floral-nav-divider" src="assets/floral_white_smaller.png" alt="floral divider">
+      <div class="header-text-content">
+        <h1>Welcome to the Food Fansite</h1>
+        <nav>
+          <ul>
+            <li><a href="index.html">Home</a></li>
+            <li><a href="pages/gallery.html">Gallery</a></li>
+            <li><a href="pages/facts.html">Facts</a></li>
+          </ul>
+        </nav>
+      </div>
+      <img class="floral-nav-divider" src="assets/floral_white_smaller.png" alt="floral divider">
+    </header>
+    <main class="content">
+      <section>
+        <h2>Explore the world of food</h2>
+        <p>Here, we share our love for all things foods-related. From yummy pictures to fun facts, this is the place for food lovers!</p>
+        <br>
```

Comment:
We want to be pretty sparing with our use of the `br` tag and try to save it for places where a line break is really meaningful to the content (Examples from the MDN docs on `br` are poems and postal addresses). It's considered better form to use CSS to set margins or create space around an element.

A primary reason to avoid `br`s is that they can make navigation with screen readers more difficult. I recommend taking a look at the accessibility concerns section of the MDN's `br` reference. Their suggested replacement is to use individual `<p>` elements and use styling on properties like the margin to create space between them. 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br#accessibility_concerns